application/src/main/java/org/togetherjava/tjbot/features/jshell/ResultRenderer.java

```diff
@@ -12,10 +12,23 @@
 
 import java.awt.Color;
 
-import static org.togetherjava.tjbot.features.jshell.render.Colors.*;
+import static org.togetherjava.tjbot.features.utils.Colors.*;
 
-public class ResultRenderer {
+/**
+ * Allows to render JShell results.
+ */
+class ResultRenderer {
 
+    /**
+     * Renders a JShell result to an embed.
+     * 
+     * @param originator the user from who to display snippet ownership
+     * @param originalCode the original code to display
```

Comment:
since nullable, what happens if null? should be mentioned in the doc

Reply:
I thought it was obvious enough, this method render stuff, so if it's null it doesn't render it.

Reply:
not obvious to me, please put it, thanks

Reply:
Changed, tell me what you think.